client/css/style.css

```diff
@@ -1498,6 +1498,14 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	outline: none;
 }
 
+[contenteditable] {
```

Comment:
Actually, make this `[contenteditable="true"]` so it's only selectable + correct cursor in edit mode.

Reply:
I don't think that's necessary in lounge?

Reply:
Why not? That's what we do everywhere else in the app except messages, i.e. UI not selectable. With this addition, it stays like that, except when the nick is being edited, when it can be selected and have the text cursor. Makes sense?